docs/cpu-target.md

```diff
@@ -112,6 +112,23 @@ When compiled into a shared library/dll - how is it invoked? The entry point is
 void computeMain(ComputeVaryingInput* varyingInput, UniformEntryPointParams* uniformParams, UniformState* uniformState);
 ```
 
+
+If compiled with `SLANG_HOST_CALLABLE` the `ISlangSharedLibrary` will export a function named `computeMain` the same name as the entry point in the original source.  
```

Comment:
I would have expected the exported name to match the original entry point declaration, but I suppose `computeMain` makes life easier for testing. Still, if we ever want to support multiple entry points they will need distinct names...

Reply:
The name does match the entry point name - which perhaps the doc above doesn't make clear enough.